### PR TITLE
completedn prep-css-flexbox exercise

### PR DESCRIPTION
**Pull request by NickHolloway**
image
image
image
image

### CODE REVIEW

File: prep-css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      min-height: 100vh;
+    }
+    #card-wrapper {
```
Comment (thebearingedge) [inline prep-css-flexbox/align-items.html:13]: Try using `.card-wrapper` to apply styles to every element with the `class` attribute of `"card-wrapper"`.

File: prep-css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc( 100%/6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html:12]: `flex-basis` only works when it is applied to children of elements with `display: flex;` try moving all of your `flex-basis` properties to the rule set for `.card-wrapper`.

File: prep-css-flexbox/justify-content.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+<html>
+<head>
+  <title>CSS Flexbox - justify-content</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-basis: 25%;
```
Comment (thebearingedge) [inline prep-css-flexbox/justify-content.html:11]: Here as well, `flex-basis` should be applied to `.card-wrapper`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-css-flexbox/align-items.html (hunk 1 above)
Comment (NickHolloway) [reply prep-css-flexbox/align-items.html:13]: image

File: prep-css-flexbox/flex-wrap.html (hunk 2 above)
Comment (NickHolloway) [reply prep-css-flexbox/flex-wrap.html:12]: image

Comment (NickHolloway) [reply prep-css-flexbox/flex-wrap.html:12]: image

File: prep-css-flexbox/justify-content.html (hunk 3 above)
Comment (NickHolloway) [reply prep-css-flexbox/justify-content.html:11]: image

File: prep-css-flexbox/align-items.html (hunk 1 above)
Comment (NickHolloway) [reply prep-css-flexbox/align-items.html:13]: image
Should display: flex; be under #card-deck or .card-wrapper?

**Pull request merged**